common/src/uint/mod.rs

```diff
@@ -41,3 +40,101 @@ pub trait BitArray {
     /// Create value representing one
     fn one() -> Self;
 }
+
+#[derive(Debug, Clone)]
+pub struct U256Encodable(pub U256);
+
+impl Encode for U256Encodable {
+    fn size_hint(&self) -> usize {
+        32
+    }
+
+    fn encode_to<T: serialization::Output + ?Sized>(&self, dest: &mut T) {
+        dest.write(&self.0.to_le_bytes());
+    }
+
+    fn encoded_size(&self) -> usize {
+        32
+    }
+}
+
+impl Decode for U256Encodable {
+    fn decode<I: serialization::Input>(input: &mut I) -> Result<Self, serialization::Error> {
+        let v = <[u8; 32]>::decode(input)?;
+        Ok(U256Encodable(U256::from_le_bytes(v)))
+    }
+}
+
+pub fn into_u512(u: U256) -> U512 {
```

Comment:
I really thought the library provided this conversion... are you sure you have to do this like that? I have a feeling there should be a conversion for this already with `into()` or `try_into()`

Reply:
Also, if we're gonna do this, there's not even close to enough tests for it.

Reply:
surprisingly the is no, expect from tuples